runtime/resource.go

```diff
@@ -0,0 +1,113 @@
+// Package runtime is a service runtime manager
+package runtime
+
+import "fmt"
+
+const (
+	TypeNamespace     = "namespace"
+	TypeNetworkPolicy = "networkpolicy"
+	TypeService       = "service"
+)
+
+// Resource represents any resource handled by runtime
+type Resource interface {
```

Comment:
It might be handy to have a String() func, which could return something in the format of "service://namespace@id:version".

Reply:
I've decided to get rid of `ID()`, and replace it with `String()`. I've implemented your suggestion of the format for the Service resource.